### DISCUSSION TITLE
Rejected for no reason?

### DISCUSSION THREAD

**Discussion by Alexandria4ever** [community/community#125547] [Programming Help]
### Body

![image](https://github.com/community/community/assets/169586770/a823ab1d-7135-4038-af8e-218df61d9f7e)
Just submitted my application for the fourth time. They just rejected me without any reason.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Skedar** [comment]
You read the documentation?

this error is telling you already submit 4 requests. this request are open and waiting for someone to answer? 

your learning community are on the github campus program?

you need to see all that first.

good luck

**Comment by RafaelJohn9** [comment]
did you apply using your school email?

**Comment by Alexandria4ever** [comment]
The previous three had reasons not to be accepted. This one mentions
literally nothing.

**Comment by Alexandria4ever** [comment]
No, don't have one. Schools don't give them out here.

**Reply by RafaelJohn9** [reply]
it would be really hard to get  the pack without a school email
A school is an institution best way to get an email:
-make requests to get it, because they have them
-Ask friend staff to assist you with theirs if they don't use them (some hardly do)

Just get the email, from there everything else will be smooth

Disclaimer: The following comment is a paraphrase or summary of information gathered from various sources and does not represent my original words. Please note that I am not the author of the following content.😁🤝

**Comment by Alexandria4ever** [comment]
You don't understand. Students not teachers are issued any school emails
here. The concept is unheard of. And anyway, I submitted alternate
documents (dated photo of ID Card) still they rejected me.

**Reply by RafaelJohn9** [reply]
it will be hard to get the pack without the school email, Just ask for your IT teacher to help you to get an email or to assist you with his/her's
